error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssortedRules.java

```diff
@@ -49,6 +49,26 @@ int after(int index, int size) {
     }
   }
 
+  /**
+   * Prefer {@link Objects#checkIndex(int, int)} over less descriptive or more verbose alternatives.
+   *
+   * <p>If a custom error message is desired, consider using Guava's {@link
+   * com.google.common.base.Preconditions#checkElementIndex(int, int, String)}.
+   */
+  static final class CheckIndexConditional {
+    @BeforeTemplate
+    void before2(int index, int size) {
```

Comment:
Nice to have added this one, thanks! We can however rename it to `before`. :ok_hand: